### PR TITLE
Open source blog

### PR DESCRIPTION
**Pull request by bobturneruk**
Best if this goes out, it's prior to Hacktoberfest. Interested particularly in @drj11's perspective on this. @annakrystalli do I need to say more about open source and reproducible research?

### CODE REVIEW

**Comment by bobturneruk** [comment]
I'll need to fix some links.

File: _posts/2020-09-21-open-source.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+date: 2020-09-21 12:00:00 UTC
+tags: 
+category:
+link:
+description:
+type: text
+excerpt_separator: <!--more-->
+---
+
+This is going to be my perspective, to some extent, but hopefully also introduces some of the legal, technical, economic and academic aspects of open source – providing context for experts and a gateway for the newly interested. *Note: I have no legal training, consult someone who does if you need to!*. But, my perspective, so we’re going back to when I was first exposed to open source, back around the millennium…
```
Comment (drj11) [inline _posts/2020-09-21-open-source.md]: Your perspective on what? You should start with something like "this is my personal perspective Open Source, what it is, how it grew, and what it means now for academic practitioners", then carry on with "introduces bblah blah blah"

File: _posts/2020-09-21-open-source.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Open Source"
+author: Robert (Bob) Turner
+slug: 2020-09-21-open-source
+date: 2020-09-21 12:00:00 UTC
+tags: 
+category:
+link:
+description:
```
Comment (drj11) [inline _posts/2020-09-21-open-source.md:10]: add an image for the social card.

File: _posts/2020-09-21-open-source.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+<!--more-->
+
+Open source was an ideological battleground. Really. The dotcom bubble had burst. Everyone had a phone, but no-one had a smartphone. If you accessed the internet is was almost certainly via a PC or laptop running Microsoft Windows.
+
+## Copyright
+
+People were beginning to realise that the internet could be used to transfer music, games and videos at scale, but there were few paid ways of doing this, so using a peer file sharing network to illegally download was increasingly commonplace. People felt OK about this, as they saw themselves as were depriving large corporations of small amounts of revenue. Said large corporations took a different stance – peer sharing was going to destroy all music, much as home taping had a generation before. Why am I banging on about this? Copyright. In much of the world, if you produce some music, or some text, or some code, for a period of time no-one is allowed to copy it without your permission. This right is something you can sell. And while copyright was something the peer-sharers ignored, it is a pillar of open source software, as we shall see.
```
Comment (drj11) [inline _posts/2020-09-21-open-source.md:25]: https://www.google.com/imgres?imgurl=https%3A%2F%2Fmemezila.com%2Fwp-content%2FNetflix-used-to-come-in-the-mail-Sure-grandma-lets-get-you-to-bed-meme-6760.png&imgrefurl=https%3A%2F%2Fmemezila.com%2FNetflix-used-to-come-in-the-mail-Sure-grandma-lets-get-you-to-bed-meme-6760&tbnid=vxHVjZjSjOZ_qM&vet=12ahUKEwj52-3PyPzrAhUNw4UKHeOVCn4QMygAegUIARCgAQ..i&docid=0KDIU3VgalcGxM&w=813&h=640&q=netflix%20used%20to%20come%20in%20the%20mail%20meme%20sure%20grandma&hl=en&ved=2ahUKEwj52-3PyPzrAhUNw4UKHeOVCn4QMygAegUIARCgAQ

**Review by drj11 — APPROVED** [review]
I like the tone and the strength of the personality that comes across. It's a little unstructured, and I'm left feeling a little bit "what's the point?", but it's fun and short. Go with it, I say.

**Comment by bobturneruk** [comment]
Thanks @drj11 - I'll do as you suggest. @davidwilby , @annakrystalli - is there more actual information I should add? I guess the point is to provide a bit of background and context for those thinking of Hacktoberfesting...

**Comment by davidwyld** [comment]
> is there more actual information I should add? I guess the point is to provide a bit of background and context for those thinking of Hacktoberfesting...

Yes a bit of segue into the vague idea of Hacktoberfest and the democratization/openness of open source contribution in your last section there wouldn't go amiss. I would just link to the events we're running and indicate what they're about. But you've got most of what you need there I guess.

**Comment by bobturneruk** [comment]
Thanks. I think the notion was that is might echo material used in the introduction to open source talk - so if there's anything (I guess, non-technical) that folks feel should go in, I'll work that in, too.

File: _posts/2020-09-21-open-source.md (hunk 1 above)
Comment (bobturneruk) [reply _posts/2020-09-21-open-source.md]: Hopefully resolved!

File: _posts/2020-09-21-open-source.md (hunk 2 above)
Comment (bobturneruk) [reply _posts/2020-09-21-open-source.md:10]: Not sure how to do this. I'll try and figure it out.

**Comment by bobturneruk** [comment]
Hopefully better now. I'd appreciate a second glance from anyone with time, but otherwise I'll put it out later today. Once I've figured out this social card thing...

File: _posts/2020-09-21-open-source.md (hunk 2 above)
Comment (drj11) [reply _posts/2020-09-21-open-source.md:10]: Looking at one of my earlier blogs: https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/blob/master/_posts/2019-08-22-learn-python.md

apparently you add an `image:` key with a `path:` sub-key.

Would have thought that picking the image would be the hard part, and sadly i'm out of inspiration.

For sure, publish without a picture if it's too annoying to find and add one.

Comment (bobturneruk) [reply _posts/2020-09-21-open-source.md:10]: Think I've figured it out by reading the readme (who knew?)

Comment (bobturneruk) [reply _posts/2020-09-21-open-source.md:10]: Sorted. I read the readme!

**Pull request merged**